client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -84,32 +86,34 @@ class ProductCreate extends React.Component {
 	}
 
 	render() {
-		const { product, className, variations, productCategories } = this.props;
+		const { siteId, product, className, variations, productCategories } = this.props;
 
 		return (
 			<Main className={ className }>
 				<SidebarNavigation />
 				<ProductHeader
 					onTrash={ this.onTrash }
 					onSave={ this.onSave }
+					saveDisabled={ ! siteId }
```

Comment:
I wonder if it might be better to utilize `onSave` for this instead of adding another prop. Right now if `onSave === undefined` the button isn't rendered at all. But what if when `onSave === false`, we display a disabled button? Then it could serve both purposes.